docs/automated_testing.md

```diff
@@ -0,0 +1,509 @@
+---
+id: automated_testing
+title: Automated Integrated Testing
+sidebar_label: Automated Testing
+slug: /automated_integrated_testing
+---
+
+<!-- original: https://docs.google.com/document/d/1hkNr78mrU3Ha98tPUL4OKWi3eNnt-1Sba7L8470u06g/edit# -->
+
+🚧 Reference for writing automated integrated tests with the Assembly Line testing framework.
+
+<!-- ## Setup
+
+For now, talk to us about it.
+
+- Have admin permissions on the repo
+- Use the setup interview (https://apps-dev.suffolklitlab.org/start/test-setup/)
+- Add this element to the page somewhere: <div data-variable="${ encode_name(str( user_info().variable )) }" id="trigger" aria-hidden="true" style="display: none;"></div>
+
+Secrets
+
+PLAYGROUND_PASSWORD
+PLAYGROUND_ID
+PLAYGROUND_EMAIL
+-->
+
+## Summary
+
+1. You write and edit `.feature` test files in your Sources folder.
```

Comment:
```suggestion
1. You write and edit `.feature` test files in your Sources folder that contain a series of steps that the test should take when filling out the interview.
```

Reply:
I really want this to be a very bare bones summary. A very quick reference. How about this?

```suggestion
1. You write and edit `.feature` test files in your Sources folder. Instructions for writing tests are below.
```

Reply:
I guess, but the issue I had with this line is that if you don't already know what a `.feature` file is, it tells you very little. Just knowing that it's some steps for the tester is good.

But this is minor for just the deep-dive version of the code. Feel free to just ignore for now.

Reply:
I think this is something that the documentation guidelines site might help with - figuring out where different pieces might go.